package.json

```diff
@@ -122,8 +121,7 @@
     "webpack": "4.19.1",
     "webpack-dev-server": "^3.1.11",
     "webpack-manifest-plugin": "2.0.4",
-    "workbox-webpack-plugin": "3.6.3",
-    "yargs": "^12.0.5"
```

Comment:
The `github-api` and `yargs` are no longer necessary since that functionality was offloaded to `github-assistant`.